src/SelfService/Configuration/Domain.cs

```diff
@@ -80,6 +80,18 @@ public static void AddDomain(this WebApplicationBuilder builder)
 
         // aad-aws-sync
         builder.Services.AddTransient<IAadAwsSyncCapabilityQuery, AadAwsSyncCapabilityQuery>();
+        
+        // azure
+        var azureResourceManifestEnabled = builder.Configuration["SS_ARM_ENABLED"];
```

Comment:
why this name `SS_ARM_ENABLED`; does not seem to indicate what it is being used for?